livre/core/render/GLContext.h

```diff
@@ -43,6 +43,11 @@ class GLContext : public std::enable_shared_from_this< GLContext >
     LIVRECORE_API void shareContext( GLContextPtr dstContextPtr );
 
     /**
+     * @return a similar context.
```

Comment:
This is handled in a cleanup commit. I did that. Renamed to ::clone() and more descriptive exp.